perf/index.js

```diff
@@ -0,0 +1,63 @@
+const levenshteinEditDistance = require('levenshtein-edit-distance');
```

Comment:
you converted all the benchmark sources to ESM but not the main file. the reason i left them CJS was because the modules we have to import can't be imported through an ES import statement. either this file should also be ESM (mashed up with require calls) or the others should be CJS